Extensions/easy_video_links.js

```diff
@@ -0,0 +1,69 @@
+
+	run: function() {
+		
```

Comment:
this is really nitpicky, but would you mind removing the blank lines at the beginning/end of functions?